features/provider/finance/invoices.feature

```diff
@@ -0,0 +1,43 @@
+@ignore-backend @javascript
+Feature: Show all provider's invoices
+  In order to manage invoices
+  As a provider
+  I want to be able to see all invoices issued by me
+
+  Scenario: Display years of invoices when there are invoices
+    Given a provider is logged in on 1st January 2011
+    And a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
+    And the provider is charging its buyers in prepaid mode
+    And an invoice of buyer "zoidberg" for January, 2011
+    And an invoice of buyer "zoidberg" for January, 2014 on 1st January 2014 (without jobs)
```

Comment:
I don't understand this what the steps above should be doing? Why is the second step called `on 1st January 2014 (without jobs)`?

Reply:
The step creates an invoice for the specific period (month+year). This `on 1st January 2014 (without jobs)` is needed so that this creation is executed on a certain date.

This is because of this validation for the invoice period :upside_down_face:  https://github.com/3scale/porta/pull/3073/files#diff-c7710675b799dc693083ef5075a5bb1fee822ddae19fe8a64a1be2f5a87bb6edR621-R628

Reply:
Ah, and "(without jobs)" is to avoid that all cron jobs execute from 1st Jan 2021 and 1st Jan 2014, which was taking too long :grimacing: